dae_RelayBoard/dae_RelayBoard_D2XX.py

```diff
@@ -39,7 +39,6 @@ def __init__(self):
         
     #currently only gets first board
     def initialise(self, *args):
-        self.disconnect()

```

Comment:
What issues were you seeing with this disconnect? On windows it makes sense to close the old handle before creating a new one.

Reply:
Well, I implemented the linux disconnect method following the same approach as the other linux methods on D2XXX file, so if no self.bb is present it raises an error when disconnecting. The issue comes when calling disconnect always when initializing the relaybox, since there is no self.bb yet, it always crashes. So maybe a check shall be added before calling disconnect inside the initialize method (to check if a device is already initialized and therefore call disconnect, or not), or maybe implement an if statement to disconnect only for windows based platforms.

Reply:
Really busy atm. Will get to this this weekend. Thanks for this btw.

Reply:
Look at the close function in FTD2XXWindows.py and implement in that manner. Basically do nothing in the close function if the self.bb is None. Remove your current exception.

If you make this change and squash all your commits, I'll test and merge if good.

Thanks.

Reply:
Ok, I have reverted the changes and removed the raise from linux driver. Could you please check it?
Many thanks!